packages/webview_flutter/webview_flutter_platform_interface/lib/src/legacy/types/web_settings.dart

```diff
@@ -125,8 +126,13 @@ class WebSettings {
   /// See also: [WebView.gestureNavigationEnabled]
   final bool? gestureNavigationEnabled;
 
+  /// Determines whether pressing a link displays a preview of the destination for the link in iOS.
+  ///
+  /// See also: [WebView.allowsLinkPreview]
+  final bool? allowsLinkPreview;
```

Comment:
I noticed this PR is also making changes in the `legacy/` folders in each package. We no longer support that version, so you shouldn't need to change anything in these folders.